tfm_ns_import.yaml

```diff
@@ -24,6 +24,10 @@
             {
                 "src": "install/image_signing/layout_files/signing_layout_s.o",
                 "dst": "targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/partition/signing_layout_s.c"
+            },
+            {
+                "src": "lib/ext/tfm_test_repo-src/app/tfm_ns_interface.c",
+                "dst": "targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/tfm_ns_interface.c"
             }
         ],
         "ARM_MUSCA_B1": [
```

Comment:
It would be nice to also import Musca B1 and S1's `region_defs.h` and `flash_layout.h` into Mbed OS (see https://github.com/ARMmbed/mbed-os/issues/14762).

The reason we didn't import this file was that the `BL2` macro in `targets.json` wasn't passed to the Arm Compiler linker script by Mbed CLI 1 (frozen scripts in mbed-os/tools/), and we manually defined this macro in `region_defs.h` as a workaround. We didn't want the import script to override our workaround. Because those files are not auto-imported, we forgot to update them and they are already outdated.

It seems better to me to define `BL2` at the top of the Arm [scatter file](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/device/TOOLCHAIN_ARMC6/musca_ns.sct) which is Mbed-only and never gets overridden by scripts. Then we can import `region_defs.h` and `flash_layout.h` untouched and never forget them. If we opt for this, the scatter file change would be part of https://github.com/ARMmbed/mbed-os/pull/15050.

Reply:
It looks like we also have other places in `mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_{B1,S1}` that use `region_defs.h`. But the CLI 1 bug only affects Arm Compiler scatter files, so we don't need to worry about other usages of this header...